client/services/Hooks.ts

```diff
@@ -99,11 +99,27 @@ function withAuthModal(isLoggedIn: boolean, cb: (...any) => any) {
   };
 }
 
+function pressEsc(ref: MutableRefObject<HTMLElement>, handler: (e: MouseEvent) => void) {
+  useEffect(() => {
+    const handleEsc = (event) => {
+      if (event.keyCode === 27) {
```

Comment:
`keyCode` is deprecated - https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Let's use `code` instead.